### PR TITLE
added text badges

### PR DESCRIPTION
**Pull request by Ankur9669**

### CODE REVIEW

File: Components/Badge/badge.css [hunk 1]
```diff
@@ -78,3 +78,26 @@
     min-width: 1.1rem;
     height: 1.1rem;
 }
+.text-badge{
+    color: var(--badge-text-color);
+    background-color: var(--badge-background-color);
+    padding: 0.3rem 0.6rem 0.3rem 0.6rem;
```
Comment (Naman-Saxena1) [inline Components/Badge/badge.css:84]: For less than 1rem, you should be using px values.

File: documentation.html [hunk 2]
```diff
@@ -375,6 +373,59 @@ <h4 class="component-heading">Badges</h4>
...
+              Example of small text badge
+              <span
+                class="text-badge text-badge-rounded-border text-badge-small"
+                >New</span
+              >
+            </p>
+            <p class="font-extra-small weight-semi-bold primary-text-color">
+              Example of very small text badge
+              <span
+                class="text-badge text-badge-rounded-border text-extra-very-small"
```
Comment (Naman-Saxena1) [inline documentation.html:417]: You can change name of text-extra-very-small to text-xx-small or some other alternative.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
Requested few changes, everything else looks fine.

File: documentation.html (hunk 2 above)
Comment (Ankur9669) [reply documentation.html:417]: Yes i will do it after i complete my CL, for now to keep things consistent i have given them same name.

File: Components/Badge/badge.css (hunk 1 above)
Comment (Ankur9669) [reply Components/Badge/badge.css:84]: But according to my resarch this would work right..

**Comment by nikhiljugale007** [comment]
Code and UI look good.

**Pull request merged**